ebodac/src/test/java/org/motechproject/ebodac/uitest/helper/TestParticipant.java

```diff
@@ -4,15 +4,88 @@
  * Created by tomasz on 30.09.15.
  */
 public class TestParticipant {
-    public String id;
-    public String name;
-    public String language;
-    public String phoneNumber;
-    public String siteId;
-    public String householdName;
-    public String headOfHousehold;
-    public String community;
-    public String address;
+    private String id;
+    private String name;
+    private String language;
+    private String phoneNumber;
+    private String siteId;
+    private String householdName;
+    private String headOfHousehold;
+    private String community;
+    private String address;
+
+    public String getId() {
```

Comment:
you added getters and setters, but somewhere in the code those properties are used directly, you have compilation errors because of this